app/src/main/java/kantwonskids/donationtrackerg14b/model/User.java

```diff
@@ -104,6 +104,42 @@ public String getPassword() {
         return password;
     }
 
+    /**
+     * Gets the role of the current user.
+     *
+     * @param role the type of account that the user should have
+     */
+    public void setUserRole(UserRole role) {
+        this.role = role;
+    }
+
+    /**
+     * Gets the role of the current user.
+     *
+     * @return the type of account that the current user has
+     */
+    public UserRole getUserRole() {
+        return this.role;
+    }
+
+    /**
+     * Sets the location where this user works (Location employees only).
+     *
+     * @param location where this user works
+     */
+    public void setLocation(Location location) {
```

Comment:
Maybe for robustness add a check to see if the user is a location employee and if not, throw an IllegalOperationException or something

Reply:
changed